lessons/8_loops/index.html

```diff
@@ -0,0 +1,308 @@
+            </li>
+             
+          </ul>
+        </section>
+  
+        <section>
+            <pre><code data-trim class="" >
+                const items = ["Item 1", "Item 2", "Item 3", "Item 4", "Item 5", "Item 6", "Item 7"];
+                //to print the items always is running items[0] -> just the index is changing
+                console.log(items[0]);  // index: 0
+                console.log(items[1]);  // index: 1
+                console.log(items[2]);  // index: 2
+                console.log(items[3]);  // index: 3
+                console.log(items[4]);  // index: 4
+                console.log(items[5]);  // index: 5
+                console.log(items[6]);  // index: 6
+            </code></pre>
+            <br/>
+            <h2 class="fragment">Solution</h2>
+            <pre class="fragment"><code data-trim class="hljs">
+                const items = ["Item 1", "Item 2", "Item 3", "Item 4", "Item 5", "Item 6", "Item 7"];
+                for (let i = 0; i < items.length; i++) {
+                    console.log(items[i]);
+                }
+            </code></pre>
+        </section>
+
+        <section>
+            <h3>For Loop structure</h3>
+            <pre><code data-trim class="hljs">
+                for (let i = 0; i < 10; i++) {
+                  // code block to be executed
+                }
+                for ([1. initialization]; [2. condition]; [3. increment]){
+                  // code block to be executed
+                }
+            </code></pre>
+            <div>
+                <ol>
+                    <li class="fragment"><b><i>let i= 0:</i></b> Initializes the counter variable i.</li>
+                    <li class="fragment"><b><i>i < 10:</i></b> Defines the condition for executing the code block.</li>
+                    <li class="fragment"><b><i>i++:</i></b> updates the loop counter (i) with a new value each time the loop runs.</li>
+                </ol>
+            </div>
+       </section>
+
+       <section>
+           <h3>Examples</h3>
+            <h4>Print numbers from 1 to 20</h4>
+           <pre><code data-trim class="hljs">
+                for (let i = 0; i < 20; i++) {
+                    console.log(i + 1)
+                }
+            </code></pre>
+            <br/>
+            <div class="fragment">
+                <h4>Print list of items</h4>
+                <pre><code data-trim class="hljs">
+                    const items = ["Item 1", "Item 2", "Item 3", "Item 4", "Item 5", "Item 6", "Item 7"];
+                    for (let i = 0; i < items.length; i++) {
+                        console.log(items[i]);
+                    }
+                </code></pre>
+            </div>
+       </section>
+
+       <section>
+           <h4>Print students' full name.</h4>
+           <pre><code data-trim class="hljs">
+                const students = [{
+                    name: "Anna",
```

Comment:
Maybe it's just me, but I would use teachers names as an example and avoid using student names

Reply:
Done!